### PR TITLE
add: pika tool for code screenshots

### PR DESCRIPTION
**Pull request by rishimohan**
Pika's code screenshot template can help create beautiful screenshots from code. It's free to use, requires no registration and comes with tons of customisation options. It also has a VSCode extension to quickly generate images from VSCode editor :)

## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1713,6 +1713,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Cronhooks](https://cronhooks.io/) - Schedule on time or recurring webhooks. Free plan allows 5 ad-hoc schedules.
   * [Hook Relay](https://www.hookrelay.dev/) - Add webhook support to your app without the hassles: done-for-you queueing, retries with backoff, and logging. The free plan has 100 deliveries per day, 14-day retention, and 3 hook endpoints.
   * [Format Express](https://www.format-express.dev) - Instant online formatter for JSON / XML / SQL.
+  * [Pika Code Screenshots](https://pika.style/templates/code-image) — Create beautiful, customizable screenshots from code snippets and VSCode using extension.
```
Comment (ripienaar) [inline README.md:1716]: Please mention "Free edition places a watermark on screenshots" or someting like that

Comment (rishimohan) [reply README.md:1716]: @ripienaar Actually this code screenshot template doesn't add any kind of watermark, even on free edition. I would like to keep it that way :)

Comment (ripienaar) [reply README.md:1716]: Maybe it doesnt, all I know is what the pricing page says so I go with that.

Screenshot 2023-02-17 at 11 43 44

Comment (rishimohan) [reply README.md:1716]: @ripienaar Yeah, you're right, it's kind of confusing. I've tried to add more clarity about the watermarks in pricing page. It looks like this now:

image

What do you think?

**Comment by ripienaar** [comment]
Thank you!

**Pull request merged**